src/ui/components/address-action/TransactionConfirmationView/TransactionConfirmationView.tsx

```diff
@@ -29,6 +33,8 @@ export function TransactionConfirmationView({
   eligibilityQuery,
   localAllowanceQuantityBase,
   onOpenAllowanceForm,
+  gasback: gasbackEstimation,
```

Comment:
Maybe we can name this `gasbackValue` and `onGasbackChange`? Like in inputs.

Reply:
I named it to be consistent with `onFeeValueReady`. It's a dirty side-effect to bubble up the value which is contained inside a child component, so it's not equivalent to a controlled input component, where value comes from the prop